cypress/e2e/awx/administration/wfApprovalsList.cy.ts

```diff
@@ -0,0 +1,595 @@
+                        cy.intercept(
+                          'POST',
+                          awxAPI`/workflow_approvals/${approval.id.toString()}/approve/`
+                        ).as('WFaction');
+                        cy.filterTableByMultiSelect('id', [approval.id.toString()]);
+                        cy.getTableRow('id', approval.id.toString(), { disableFilter: true })
+                          .within(() => {
+                            cy.getByDataCy('actions-column-cell').within(() => {
+                              cy.getByDataCy('approve').click();
+                            });
+                          })
+                          .then(() => {
+                            cy.actionsWFApprovalConfirmModal('approve');
+                          });
+                        cy.wait('@WFaction')
+                          .its('response')
+                          .then((response) => {
+                            expect(response?.statusCode).to.eql(204);
+                          });
+                        cy.reload();
+                        cy.getByDataCy('status-column-cell').should('have.text', 'Approved');
+                        cy.intercept(
+                          'DELETE',
+                          awxAPI`/workflow_approvals/${approval.id.toString()}/`
+                        ).as('deleteWFA');
+                        cy.waitForWorkflowJobStatus(response.id.toString());
+                        cy.getByDataCy('actions-column-cell').within(() => {
+                          cy.clickKebabAction('actions-dropdown', 'delete-workflow-approval');
+                        });
+                        cy.actionsWFApprovalConfirmModal('delete');
+                        cy.wait('@deleteWFA')
+                          .its('response')
+                          .then((response) => {
+                            expect(response?.statusCode).to.eql(204);
+                          });
+                      }
+                    );
+                  });
+                });
+            });
+          }
+        );
+        cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+      });
+    });
+
+    it('admin can deny and then delete a workflow approval from the list row item', () => {
+      cy.createAwxWorkflowJobTemplate({
+        name: 'E2E Workflow Approval-DENY-' + randomString(4),
+        organization: organization.id,
+        inventory: inventory.id,
+      }).then((wfjt) => {
+        workflowJobTemplate = wfjt;
+        cy.createAwxWorkflowVisualizerJobTemplateNode(workflowJobTemplate, jobTemplate).then(
+          (jtNode) => {
+            jobTemplateNode = jtNode;
+            cy.createAwxWorkflowVisualizerApprovalNode(workflowJobTemplate).then((appNode) => {
+              approvalWFNode = appNode;
+              cy.createWorkflowJTAlwaysNodeLink(approvalWFNode, jobTemplateNode);
+
+              cy.visit(
```

Comment:
We should not be using `cy.visit` and we should not be hard coding url paths in the the tests